src/backend/distributed/commands/extension.c

```diff
@@ -729,10 +729,15 @@ IsDropCitusStmt(Node *parseTree)
 	{
 		return false;
 	}
+	DropStmt *dropStmt = (DropStmt *) parseTree;
```

Comment:
minor:

```suggestion

    /* check if the drop command is a DROP EXTENSION command */
	DropStmt *dropStmt = (DropStmt *) parseTree;
```

Reply:
Actually we can maybe rename the method as `IsDropCitusExtensionStmt`, what do you think?

Reply:
`IsDropCitusExtensionStmt` would be a really better function name

Reply:
@onurctirtir, updated some docs as well, does it look good now?

Reply:
yes